library-versions/README.mkd

```diff
@@ -10,6 +10,7 @@ of the GHC wiki.
 1. First dump the compiler's global package database: `ghc-pkg list --global > tmp`
 2. Feed this to `pack_pkg_list`: `./pack_pkg_list.hs tmp`
 3. Insert the output of `pkg_pkg_list` into `pkg_versions.txt`
-4. Use `mk_pkg_table` to generate the table: `./mk_pkg_table.hs < pkg_versions.txt`
+4. Manually add the `Win32` version at the top of `pkg_versions.txt`
```

Comment:
A script for this step would be nice. This time I just navigated the GHC repo starting at the tag for the respective GHC version.

Reply:
Indeed.